backend/tests/clubs/test_views.py

```diff
@@ -3,6 +3,7 @@
 import json
 import os
 from collections import Counter
+from unittest import mock
```

Comment:
Is this needed for anything other than `mock.patch()`? If so we already import that

Reply:
Good catch, thanks!